docs/adrs/00005-license-export-analytics.md

```diff
@@ -0,0 +1,126 @@
+# 00005. Analysis of the license data structure.
+
+Date: 2025-01-23
+
+## Status
+
+DRAFT
+
+## Context
```

Comment:
Why not a few paragraphs about what and why this is being done ? It just launches into a 'laundry' list of formats ... prepare the reader